.github/workflows/codeql.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+      - '[1-9]+.[0-9]+.x'
```

Comment:
I thinks it should be configured starting branch 3.0.x and above

Reply:
@puneetbehl Please see my [other comment](https://github.com/grails/grails-quartz/pull/124/files/ba15cadfdd081bc1f5bb0c655ec5dd6e928cc68f#r1500896961) about this.